ui/src/message_popup/author/questions_page.jsx

```diff
@@ -59,18 +66,37 @@ export default React.createClass({
     return questionString.split(text).length;
   },
   
+  onQuestionsReceived(err, response){
+    const allDatabaseQuestions = JSON.parse(response.text).row;
+    if(allDatabaseQuestions === undefined) {
```

Comment:
That's a common pattern for loading data from the server - have one "container" component handle fetching data, tracking that state, and then branching between a loading state a "pure" UI component that always requires all the props and handles showing the data to the user.